Demo/ImagePickerDemo/ImagePickerDemo/Info.plist

```diff
@@ -24,6 +24,8 @@
 	<true/>
 	<key>UILaunchStoryboardName</key>
 	<string>LaunchScreen</string>
+	<key>UIMainStoryboardFile</key>
```

Comment:
`Ever` might have come of as a bit dramatic but I have yet to drink the cool-aid. I just think it makes it harder to collaborate as it screws up all the pull requests.